PSKoans/Koans/dbatools/AboutNewDatabases.ps1

```diff
@@ -0,0 +1,194 @@
+#Requires -Modules dbatools
+using module PSKoans
+[Koan(Position = 1003)]
+param()
+<#
+    New-DbaDatabase
+
+    Getting details and querying a database are useful when there is a database to query.
+    What if we need to create a new database?
+    dbatools has us covered with New-DbaDatabase
+#>
+Describe 'New-DbaDatabase' {
+
+    <#
+        Let's set up the environment for you.
+        
+        Unless you want the Koans to nearly always fail, I would suggest not messing with this bit.
+    #>
+    Mock -CommandName New-DbaDatabase -MockWith {
+        [PSCustomObject]@{
+            ComputerName = 'localhost'
+            InstanceName = 'MSSQLSERVER'
+            SqlInstance = 'localhost'
+            Name = 'random-123456789'
+            Status = 'Normal'
+            IsAccessible = $true
+            RecoveryModel = 'Simple'
+            LogReuseWaitStatus = 'Nothing'
+            SizeMB = 16
+            Compatibility = 'Version140'
+            Collation = 'SQL_Latin1_General_CP1_CI_AS'
+            Owner = $ENV:USERNAME
+            LastFullBackup = (Get-Date '0001-01-01')
+            LastDiffBackup = (Get-Date '0001-01-01')
+            LastLogBackup = (Get-Date '0001-01-01')
```

Comment:
```suggestion
            LastFullBackup = Get-Date '0001-01-01'
            LastDiffBackup = Get-Date '0001-01-01'
            LastLogBackup = Get-Date '0001-01-01'
```

Reply:
There's a couple of these :)

Reply:
Resolved in this file - I'll check out the others